### PR TITLE
Correct moveit logo path in readme.md file

### LINKED ISSUE #2946: MoveIt logo at top of README is a broken link
The MoveIt logo, which is the first thing in this repo's README, seems to be a broken link now.

### PR DESCRIPTION
**Pull request by akky20**
### Description

Fix #2946 
Added a different url path for the image

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/moveit/moveit2/blob/main/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://moveit.picknik.ai/humble/doc/examples/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/moveit/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by sea-bass — DISMISSED** [review]
LGTM, thanks for chasing down the image!

Weirdly enough, the original link still works... who knows why it didn't render anymore?
http://moveit.ros.org/assets/logo/moveit_logo-black.png

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="https://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
+<img src="https://moveit.picknik.ai/main/_static/rolling-small.png" alt="MoveIt Logo" width="200"/>
```
Comment (rhaschke) [inline README.md:1]:
```suggestion
<img src="http://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
```

**Review by rhaschke — COMMENTED** [review]
> Weirdly enough, the original link still works... who knows why it didn't render anymore?

The original link is a secure link: 
https://moveit.ros.org/assets/logo/moveit_logo-black.png, which doesn't work anymore because the certificate of the site is invalid. The normal HTTP link works.

The new link changes the image from black to white. Is this intended?
| old | new |
|--|--|
| old | new |

**Comment by sea-bass** [comment]
> The new link changes the image from black to white. Is this intended?

Hah, I didn't realize this because I use dark mode and the white actually looks "normal" on my screen. Good catch!

Ideally, a white background with black text would look best for all

**Comment by akky20** [comment]
@sea-bass @rhaschke Yes , I have corrected it

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<img src="https://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
+<img src="https://camo.githubusercontent.com/24beba601ee2b6d2251cc0ef1a5abe5619af4097ec3495cdee1f5a53fdbdf8dd/687474703a2f2f6d6f766569742e726f732e6f72672f6173736574732f6c6f676f2f6d6f766569745f6c6f676f2d626c61636b2e706e67" alt="MoveIt Logo" width="200"/>
```
Comment (rhaschke) [inline README.md:1]: This is definitely not a sustainable link. Why didn't you used the one I proposed?
```suggestion
<img src="http://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
```

**Review by rhaschke — CHANGES_REQUESTED** [review]

**Comment by akky20** [comment]
@sea-bass @rhaschke Yes , I have corrected it please have a look

File: README.md (hunk 2 above)
Comment (akky20) [reply README.md]: sorry for the misunderstanding , I have corrected it

**Review by sea-bass — APPROVED** [review]

**Pull request merged**